app/models/offence.rb

```diff
@@ -75,6 +84,6 @@ def pleas_array
   def allocation_decisions
     return [] if details.blank?
 
-    details.flat_map { |detail| detail["allocationDecision"] }
+    details.flat_map { |detail| detail["allocationDecision"] }.uniq.compact
```

Comment:
I believe this `uniq.compact` needs to go on the mode_of_trial_reasons block at line:46. 

This is because it may be possible for an  allocation decision to be unique du to ids, as below, but its code and description to not be

```
                "allocationDecision": {
                  "originatingHearingId": "75865225-3608-4715-a800-c8b001a7f6d2",
                  "offenceId": "a588aefd-006a-41d9-a42d-3beb450cbc0f",
                  "allocationDecisionDate": "2020-08-26",
                  "motReasonId": "f8eb278a-8bce-373e-b365-b45e939da38a",
                  "sequenceNumber": 30,
                  "motReasonDescription": "Defendant elects trial by jury",
                  "motReasonCode": "04"
                },
```